packages/components/src/external-link/index.tsx

```diff
@@ -65,18 +63,16 @@ function UnforwardedExternalLink(
 			target="_blank"
 			rel={ optimizedRel }
 			ref={ ref }
+			style={ { textDecoration: 'none' } }
 		>
-			{ children }
+			<span style={ { textDecoration: 'underline' } }>{ children }</span>
```

Comment:
Also, this will break a bunch of tests that expect the link text to be a direct child of the link.

Reply:
`span`s inside `a` is valid html. So this would just require the tests to be updated, right?

Reply:
Tests are failing due to a string mismatch at the accessibility tree level, so it should be fixed by https://github.com/WordPress/gutenberg/pull/60255#discussion_r1543503197, I think. (It's good that they failed and flagged the issue 🙂)

Reply:
Yes, after addressing Lena's abovementioned comment, things should be good.